.github/workflows/ci.yml

```diff
@@ -3,13 +3,45 @@ name: Node CI
 on: [push, pull_request]
 
 jobs:
+  validate-yarn-cache:
```

Comment:
This step ensures the test jobs later will always be served with proper yarn cache.

Reply:
I'm not sure it'll be in the default distribution. One problem with the v1 was that installs could be done in various slightly different ways that made the surface for unforeseen edge cases much larger. I'd like to avoid that with one standardized path as much as possible.

That being said, a plugin to do this would be trivial, so we probably can find something eventually, once we've figured out a good way to develop/host such plugins (it's already possible, but there's no comprehensive documentation).

Reply:
You can sort of get this if you set this specific job to install using PnP, which links so fast you shouldn't really notice.
Set the env variable `YARN_NODE_LINKER=pnp`

Reply:
@merceyz We opt-out of PnP because Flow does not support PnP: https://github.com/yarnpkg/berry/issues/634#issuecomment-565582736.

Reply:
Indeed, but for this specific job, validate-yarn-cache, its job is to generate a valid cache, not run Flow, so installing using PnP would do that without the cost of linking. The cache for PnP and node-modules is the same.

Reply:
Awesome: https://github.com/babel/babel/pull/11781/checks?check_run_id=991006280#step:6:27 The pnp linker costs only 4s compared to 20s of `node-modules`.

Reply:
You can reduce it even more if you also set `YARN_ENABLE_SCRIPTS=false`, you can do this since you don't need to run postinstalls to get a valid cache

Reply:
Impressive: https://github.com/babel/babel/pull/11781/checks?check_run_id=991025473#step:6:26 3s on the linker stage now. @merceyz Thank you!